src/components/StyleGuide/ColorCard.tsx

```diff
@@ -469,18 +469,26 @@ export const cssVars = {
     },
     {
       name: "ui.warning.primary",
-      dataBlackColor: checkContrast(9.24),
-      dataWhiteColor: checkContrast(2.27),
-      dataBgPageColor: checkContrast(2.27),
-      dataBgDefaultColor: checkContrast(2.08),
+      dataBlackColor: checkContrast(8.53),
```

Comment:
Where did these updated numbers come from?

Reply:
These are the updated contrast ratio values based on the new hex color.  I use a plugin in Figma to generate the values.